Loadouts/blumsv_brit_l85_ddpm.hpp

```diff
@@ -0,0 +1,318 @@
+#define SAM_MAG "rhs_fim92_mag:3"
+#define SAM_MAG2 "rhs_fim92_mag:2"
+// Sniper Rifle
+#define SNIPER "CUP_srifle_L129A1"
+#define SNIPER_MAG "20Rnd_762x51_Mag:10"
+#define SNIPER_ATTACHMENTS "optic_LRPS","CUP_acc_ANPEQ_2","CUP_bipod_Harris_1A2_L"
+// Spotter Rifle
+#define SPOTTER "CUP_arifle_L85A2_G"
+#define SPOTTER_MAG RIFLE_MAG
+#define SPOTTER_ATTACHMENTS "CUP_optic_ACOG","CUP_acc_ANPEQ_2"
+// SMG
+#define SMG "CUP_smg_MP5A5"
+#define SMG_MAG "CUP_30Rnd_9x19_MP5:6"
+// Pistol
+#define PISTOL "CUP_hgun_Glock17_tan"
+#define PISTOL_MAG "CUP_17Rnd_9x19_glock17:3"
+// Grenades
+#define LEADER_GRENADES BASE_FRAG,LEADER_SMOKES,SIDE_CHEM_LIGHT
+// Gear
+#define TOOLS BASE_TOOLS
+#define LEADER_TOOLS BASE_LEADER_TOOLS,SIDE_KEY
+#define LINKED BASE_LINKED
+#define LEADER_LINKED BASE_LEADER_LINKED
+
+class Car {
+  TransportWeapons[] = {AT};
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Tank {
+  TransportWeapons[] = {AT};
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Helicopter {
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Plane {};
+class Ship_F {};
+
+class potato_usmc_rifleman {// rifleman
+  uniform[] = {"CUP_U_B_BAF_DDPM_S2_UnRolled","CUP_U_B_BAF_DDPM_S1_RolledUp"};
+  vest[] = {"CUP_V_BAF_Osprey_Mk2_DDPM_Soldier1","CUP_V_BAF_Osprey_Mk2_DDPM_Soldier2"};
+  headgear[] = {"CUP_H_BAF_Helmet_1_DDPM","CUP_H_BAF_Helmet_3_DDPM"};
+  backpack[] = {"B_AssaultPack_cbr"};
+  backpackItems[] = {BASE_MEDICAL};
+  weapons[] = {RIFLE};
+  magazines[] = {RIFLE_MAG,BASE_GRENADES};
+  items[] = {TOOLS};
+  linkedItems[] = {LINKED};
+  attachments[] = {RIFLE_ATTACHMENTS};
+  opticChoices[] = {ALT_OPTICS};
+};
+class Fic_Soldier_Carbine: potato_usmc_rifleman {// carbine-man
+  weapons[] = {CARBINE};
+  magazines[] = {CARBINE_MAG,BASE_GRENADES};
+};
+class Soldier_TL_F: potato_usmc_rifleman {// FTL
+  vest[] = {"CUP_V_BAF_Osprey_Mk2_DDPM_Grenadier"};
+  weapons[] = {GLRIFLE};
+  magazines[] = {GLRIFLE_MAG,GLRIFLE_MAG_HE,GLRIFLE_MAG_SMOKE,LEADER_GRENADES};
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,BINOS};
+};
+class potato_usmc_sl: Soldier_TL_F {// SL
+  vest[] = {"CUP_V_BAF_Osprey_Mk2_DDPM_Officer"};
+  backpack[] = {"B_Kitbag_cbr"};
+  handguns[] = {PISTOL};
+  magazines[] += {PISTOL_MAG};
+  linkedItems[] = {LINKED,LEADER_LINKED,RANGE_FINDER};
+  items[] += {RADIO_MR};
+};
+class potato_usmc_coy: potato_usmc_sl {// CO
+  items[] += {RADIO_LR};
+};
+class potato_usmc_xo: potato_usmc_coy {// XO/PL Sgt
+};
+class potato_usmc_sgt: potato_usmc_rifleman {
+  backpack[] = {SIDE_UAV_BACKPACK};
+  linkedItems[] += {SIDE_UAV_TERMINAL};
```

Comment:
Is this typical? To have a UAV terminal? I can't remember the pattern.

Reply:
This is the COY UAV operator, so yes. Potato: USMC doesnt have a UAV operator unit, so I had to make do. I've added a comment to that effect.